moler/device/textualdevice.py

```diff
@@ -251,21 +254,29 @@ def from_named_connection(cls, connection_name):
     def __del__(self):
         self._stop_prompts_observers()
 
-    def _collect_cmds_for_state_machine(self):
-        for state in self._get_available_states():
-            self._cmdnames_available_in_state[state] = dict()
+    def _load_cmds_for_state(self, state):
```

Comment:
I would change name to `_load_cmdnames_for_state` since current name is missleading. When I read it inside `_create_cmd_instance()` I thought it is loading all command objects. Only after jumping into method body I have realized "Oh no, it loads just names"

Reply:
Done.